blue_bringup/launch/include/run.launch.xml

```diff
@@ -3,11 +3,17 @@
   <!-- What kind of robot am I? -->
   <arg name="configuration" default="full" doc="left, right, or full" />
   <arg name="version" default="2" doc="1 or 2" />
+  <arg name="on_stand" default="false" doc="true or false" />
```

Comment:
Also, I know I wrote this originally but maybe the doc string could be more descriptive (sorry!)

Reply:
Yes, that makes sense. I'll update that.

Reply:
Updated! Let me know what you think.

Reply:
thanks!